### PR TITLE
Add one unit test to account function

### PR DESCRIPTION
**Pull request by sevazhidkov**

### CODE REVIEW

**Comment by sudheesh001** [comment]
@sevazhidkov The easiest would be to write a test for the `hello()` function. You can use `assert` to assert the response that you're getting. It'd always be the same.

**Comment by sevazhidkov** [comment]
Sorry, didn't see message in google code-in portal

**Comment by sudheesh001** [comment]
@sevazhidkov Reposting this, Maybe this should help get you started

----
Okay you can do the following.
1. Move this current `test.py` file to `sampleUsage.py`
2. Create a new `test.py` file and `import unittest`
3. Write a simple test in `test.py` as follows, here's an example for you to experiment with
```python
from loklak import Loklak
import unittest

class loklakTest(unittest.TestCase):

	def testHello(self):
		l = Loklak().hello()
		self.assertEqual(l['status'],'ok')

if __name__ == '__main__':
    unittest.main()
```
You can directly use this, but You need to write another test for this for any other function before I can accept your submission.

**Comment by sevazhidkov** [comment]
I almost do it minute before :) Ok, I will write another test

File: loklaktest.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import unittest
+from loklak import Loklak
+
+
+class TestLoklakHelloAPI(unittest.TestCase):
+    def setUp():
+        self.loklak = Loklak()
```
Comment (sudheesh001) [inline loklaktest.py]: What are you trying to do here ? What does this setup mean ?
```
======================================================================
ERROR: testHello (__main__.loklakTest)
----------------------------------------------------------------------
TypeError: setUp() takes no arguments (1 given)

----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)
```

File: loklaktest.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import unittest
+from loklak import Loklak
+
+
+class TestLoklakHelloAPI(unittest.TestCase):
```
Comment (sudheesh001) [inline loklaktest.py]: Create a genetic class called `LoklakTest` and use the `HelloTest` inside it.

File: loklaktest.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    def setUp(self):
+        self.loklak = Loklak()
+
```
Comment (sudheesh001) [inline loklaktest.py]: What is this test accomplishing ? `setUp` function ?

**Comment by sevazhidkov** [comment]
Is it ok, should I create new test?

**Comment by sudheesh001** [comment]
@sevazhidkov What is the output you're getting while running the tests ?
```
E
======================================================================
ERROR: test_hello (__main__.TestLoklak)
----------------------------------------------------------------------
TypeError: test_hello() takes no arguments (1 given)

----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)
```
Here's mine, this code needs to be fixed as follows
```python
import unittest
from loklak import Loklak

class TestLoklak (unittest.TestCase):
    def setUp(self):
        self.loklak = Loklak()
    
    def test_hello(self):
        result = self.loklak.hello()
        self.assertEqual(result, "{'status': 'ok'}")

if __name__ == '__main__':
    unittest.main()
```
But I've purposely left another error in this for you to fix. Here's the error:
```
F
======================================================================
FAIL: test_hello (__main__.TestLoklak)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "testLoklak.py", line 10, in test_hello
    self.assertEqual(result, "{'status': 'ok'}")
AssertionError: {u'status': u'ok'} != "{'status': 'ok'}"

----------------------------------------------------------------------
Ran 1 test in 0.518s

FAILED (failures=1)
```

**Comment by sevazhidkov** [comment]
Solved

**Comment by sudheesh001** [comment]
@sevazhidkov So far so good, but the `hello()` test is what i've provided you, Could you write a test for `peers()` ? Your test for the peers should be 
1. to check if the `peers []` exists 
2. `count` has to be greater than equal to 0

Here's a sample response you'd get
```python
{
  "peers" : [ {
    "class" : "HelloServlet",
    "host" : "14.139.85.200",
    "port.http" : -1,
    "port.https" : -1,
    "lastSeen" : 1449511359350,
    "lastPath" : "/api/hello.json",
    "peername" : "anonymous"
  }, {
    "class" : "HelloServlet",
    "host" : "85.25.192.210",
    "port.http" : 9000,
    "port.https" : 9443,
    "lastSeen" : 1449508322626,
    "lastPath" : "/api/hello.json",
    "peername" : "anonymous"
  } ],
  "count" : 2
}
```
Finishing this would mean the end of the task.

**Comment by sevazhidkov** [comment]
Ok

**Comment by sevazhidkov** [comment]
Done

**Comment by sudheesh001** [comment]
@sevazhidkov Very well done. Now one small thing, For one issue, we maintain one commit. All this to add a unit test has to be done in one single commit. Can you `squash` your code to maintain only one commit ?
Here's an asciinema on rebasing and squashing that should help you out.
https://asciinema.org/a/11269

You're almost there. Keep up the good work.

**Comment by sudheesh001** [comment]
Oh just a tip, make sure when you push to the branch after rebasing, you do a `git push origin patch-1 -f` so that you don't read the errors on pull before push from git

**Comment by sevazhidkov** [comment]
Please look at my commits history, what I'm doing wrong?

**Comment by sevazhidkov** [comment]
Found my mistake. Thank you very match.

**Comment by sudheesh001** [comment]
@sevazhidkov Great work. I am merging this. `r+` Clean and well written tests for the peers and you've written more than what I expected.

**Pull request merged**